### PR TITLE
wc: add omit filename example

### PR DESCRIPTION
**Pull request by stsourlidakis**
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/wc.md [hunk 1]
```diff
@@ -17,3 +17,7 @@
 - Count characters in file (taking multi-byte character sets into account):
 
 `wc -m {{file}}`
+
+- Omit the filename from the output:
```
Comment (schneiderl) [inline pages/common/wc.md]: Maybe this will read better?

```suggestion
- Count words in a file, omitting the filename from the output:
```

**Comment by stsourlidakis** [comment]
My example intents to explain how to omit the filename and not what the `-l` flag does, maybe I should remove the flag to make it clearer.
Ommiting the filename is useful when `wc` is used as a subcommand e.g. `echo "The file contains $(wc -l < file) lines"`

**Comment by agnivade** [comment]
I understand, but this is not really in the scope of tldr pages. The tldr project is more of an explainer than a cheatsheet -- in other words, the underlying mission is not to merely provide copy-pastable snippets (like, say, commandlinefu.com) but rather to allow readers to understand the language of the commands, so that the logic behind the options is evident and they can then derive new commands from the provided examples, or compose several simple examples into more complex ones that are useful to them.

**Comment by stsourlidakis** [comment]
I understand, feel free to close the PR.
Thank you both for taking time to review this!

**Comment by agnivade** [comment]
No problem.

**Pull request closed without merging**